scripts/CRLP_TEST_VALIDATE_ROLLUPS.cls

```diff
@@ -519,6 +557,33 @@ private static Boolean ALLOC_INCLUDES_MEMBERSHIP = true;
         }
         insert allocs;
 
+        // Create Account Soft credits for all Opps except the Organization one and the Membership ones
+        Account accForSoftCredits = [SELECT Id FROM Account WHERE Name = :ACCOUNT_NAME_ASC LIMIT 1];
+        Id membershipRTId = UTIL_RecordTypes.getRecordTypeId(Opportunity.SObjectType, 'Membership');
+        for (Integer i=0; i<opps.size(); i++) {
+            Opportunity opp = opps[i];
+            if (opp.AccountId != orgAcct.Id
+                    && opp.RecordTypeId != membershipRTId
+                    && opp.StageName == closedWonStage) {
+                Account_Soft_Credit__c accSC = new Account_Soft_Credit__c(
+                        Account__c = accForSoftCredits.Id,
+                        Opportunity__c = opp.Id,
+                        Amount__c = ACCOUNT_SOFT_CREDIT_AMOUNT,
+                        Role__c = 'Influencer'
```

Comment:
@bethbrains  Might be best to put this in a constant at the top of the class. Also, is there a Filter Rule that includes or excludes this specific Role?

Reply:
Done, and there is now!